### PR TITLE
Update Examples w/ stripe-samples

### PR DESCRIPTION
**Pull request by yuki-stripe**
## Summary
- Point to stripe-samples, new guides in README
- Delete duped examples in Custom Integration in favor of `stripe-samples`

## Motivation
`stripe-samples` contains examples for:
* [PaymentIntents automatic confirmation](https://github.com/stripe-samples/accept-a-card-payment/tree/master/using-webhooks)
* [PaymentIntents manual confirmation](https://github.com/stripe-samples/accept-a-card-payment/tree/master/without-webhooks)
* [SetupIntents](https://github.com/stripe-samples/mobile-saving-card-without-payment)

## Testing
Ran Custom Integration

### CODE REVIEW

File: Example/README.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
-- [**Custom Integration** Example/Custom Integration/README.md](/Example/Custom%20Integration/README.md)
+## [**Basic Integration**](/Example/Basic%20Integration/README.md)
+<p align="center">
+<img src="https://raw.githubusercontent.com/stripe/stripe-ios/a87e2fb12ce1ba6b45a075ee22e0da5072a54279/standard-integration.gif" width="240" alt="STPPaymentCardTextField" align="center">
+</p>
+
+This example app demonstrates how to build a payment flow using our prebuilt UI component integration. 
+ 
+ 
+## [**Custom Integration** Example/Custom Integration/README.md](/Example/Custom%20Integration/README.md)
```
Comment (yuki-stripe) [inline Example/README.md]: There's probably a better name for Custom Integration.  It's more like 'APM Examples'.  

"Non-card Payment Examples"?
"Other Payment Methods"?

Comment (csabol-stripe) [reply Example/README.md]: I like "Non-Card Payment Examples"

Comment (csabol-stripe) [reply Example/README.md]: Okay with keeping "Custom Integration" for now as well

**Comment by davidme-stripe** [comment]
RIP, I just used these today to debug a change to card tokenization in Stripe.framework. :cry:

**Pull request merged**